ui/apps/platform/src/services/DeploymentsService.ts

```diff
@@ -52,12 +44,51 @@ export function fetchDeployments(
     if (query) {
         queryObject.query = query;
     }
-    const params = queryString.stringify(queryObject, { arrayFormat: 'repeat', allowDots: true });
+    return queryString.stringify(queryObject, { arrayFormat: 'repeat', allowDots: true });
+}
+
+/**
+ * Fetches list of registered deployments.
+ *
+ * Changes from the 'fetchDeploymentsLegacy' function:
+ * - uses the new `SearchFilter` type instead of `RestSearchOption`
+ * - Does not fetch process information linked to the deployment
+ */
+export function listDeployments(
+    searchFilter: SearchFilter,
+    sortOption: Record<string, string>,
+    page: number,
+    pageSize: number
+): Promise<ListDeployment[]> {
+    const params = fillDeploymentSearchQuery(searchFilter, sortOption, page, pageSize);
+    return axios
+        .get<{ deployments: ListDeployment[] }>(`${deploymentsUrl}?${params}`)
+        .then((response) => response?.data?.deployments ?? []);
+}
+
+/**
+ * Fetches list of registered deployments.
+ *
+ * Changes from the 'legacy' version of this same function:
+ * - returns a 'cancel' function to abort the request
+ * - uses the new `SearchFilter` type instead of `RestSearchOption`
+ * - Does not implicitly read the value of "shouldHideOrchestratorComponents"
+ */
+export function fetchDeployments(
```

Comment:
Perhaps this should be named as `fetchDeploymentsWithProcessInfo` - otherwise for user of `DeploymentService` it can be confusing as what to use - `fetchDeployments` or `listDeployments`